### PR TITLE
update latest common

### PR DESCRIPTION
**Pull request by viennaa**

### CODE REVIEW

File: openstack/maia/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Expose Prometheus as multi-tenant OpenStack service
 name: maia
-version: 1.4.14
+version: 1.4.15
```
Comment (richardtief) [inline openstack/maia/Chart.yaml:4]: The script is too ambitious.

**Review by richardtief — APPROVED** [review]
The charts have been bumped up twice. Looks like the script does it once each for Thanos and Prometheus.

File: system/vmware-monitoring/Chart.yaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
 apiVersion: v2
 name: vmware-monitoring
-version: 1.5.6
+version: 1.5.8
```
Comment (richardtief) [inline system/vmware-monitoring/Chart.yaml:3]: Double bump.

Comment (viennaa) [reply system/vmware-monitoring/Chart.yaml]: all addressed

**Review by richardtief — APPROVED** [review]

**Pull request merged**